e2e/paths/enterInternationalElements_test.js

```diff
@@ -0,0 +1,29 @@
+const config = require('../config.js');
+
+Feature('Enter international elements');
+
+Before((I, caseViewPage) => {
+	I.logInAndCreateCase(config.localAuthorityEmail, config.localAuthorityPassword, config.eventSummary, config.eventDescription);
+	I.waitForElement('.tabs', 10);
+	caseViewPage.goToNewActions(config.applicationActions.enterInternationalElement);
+	I.waitForElement('ccd-case-edit-page', 10);
+});
+
+Scenario('test form save with no details entered', (I) => {
```

Comment:
do you need the word test at the beginning of your scenarios?

Reply:
would also reword this slightly. You aren't testing the form is saved you're testing the c110a application is updated